src/lib/constants.js

```diff
@@ -57,6 +74,12 @@ export const API_ERROR = {
   NOT_FILE_UPLINK: 'file doesn\'t exist on uplink',
   MAX_USERS_REACHED: 'maximum amount of users reached',
   VERSION_NOT_EXIST: 'this version doesn\'t exist',
```

Comment:
Here also, `this version does not exist`

Reply:
Same problem above.